python/examples/integrations/Pyspark_Profiling.ipynb

```diff
@@ -659,7 +659,7 @@
    "hash": "16a10773934acde374a1cd808bcd53b1085f60e17ec18f4c0c26564dd890a5a0"
   },
   "kernelspec": {
-   "display_name": "Python 3.9.12 ('.venv': venv)",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
@jamie256 is this something we want to change for everyone?

Reply:
It's fine, the kernelspec display_name will pick up the last kernel the notebook was run on if you are editing and testing it. The previous value wasn't intentional so much as incidental to the developer workflow of the previous authors.

But I'm guessing this file doesn't need any edits for this PR?